recipes/protlearn/meta.yaml

```diff
@@ -0,0 +1,84 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "protlearn" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tadorfer/protlearn/archive/v0.0.1.tar.gz
```

Comment:
Can we use the package tarball from pypi. If so, use the canonical form:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14